core/templates/pages/exploration-player-page/layout-directives/progress-nav.component.ts

```diff
@@ -38,14 +37,14 @@ import { SchemaFormSubmittedService } from 'services/schema-form-submitted.servi
   templateUrl: './progress-nav.component.html'
 })
 export class ProgressNavComponent {
-  @Input() isLearnAgainButton: () => void;
+  @Input() isLearnAgainButton: boolean;
   @Input() displayedCard: StateCard;
   @Input() submitButtonIsShown: boolean;
-  @Input() submitButtonIsDisabled: boolean;
   @Output() submit: EventEmitter<void> = (
     new EventEmitter());
   @Output() clickContinueButton: EventEmitter<void> = (
     new EventEmitter());
+  @Output() changeCard: EventEmitter<number> = new EventEmitter();
```

Comment:
[Not a blocking comment] How do we name emitters generally in the codebase? Do we prefer them to start with verbs (like function names) or should they be something like "onCardChanged" or similar?

Reply:
Angular style guide discourages use of on prefix with Output properties.

https://angular.io/guide/styleguide#dont-prefix-output-properties

Reply:
Ah, I see, so looks like it will be named like functions then.